content-src/experiments/price-wise.yaml

```diff
@@ -0,0 +1,47 @@
+---
+addon_id: shopping-testpilot@mozilla.org
+bug_report_url: "https://github.com/mozilla/price-wise/issues"
+contributors:
+  -
```

Comment:
no other contributors?

Reply:
Nope, asked for volunteers/victims in a few meetings and nobody seemed interested.

Reply:
also there's a role field...we skipping that?